### PR TITLE
SwiftUI View: Rounded rectangle shape

### PR DESCRIPTION
**Pull request by cilippofilia**
### Description
Adding new SwiftUI Entry about rounded rectangle shape  view

### Type of Change
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`RoundedRectangle`** view is a pre-built rectangular shape with rounded corners that is aligned inside the frame of the `View` containing it, and can be visually modified as desired.
+
+SwiftUI has several built-in shapes, such as a rectangle, ellipse and capsule.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md:19]: Add links to the other shape entries.

File: content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Rounded Rectangle'
```
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md:2]:
```suggestion
Title: 'RoundedRectangle'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cilippofilia I did a first review. In addition to the other comments, can you change the file path/name from `roundedRectangle/roundedRectangle` to `roundedrectangle/roundedrectangle`

File: content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md (hunk 1 above)
Comment (cilippofilia) [reply content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md]: I don't know how to link shapes that are in a PR. can you help me out please?

Comment (cilippofilia) [reply content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md]: I am trying to change it but results in an error. Any idea why?
Screenshot 2022-12-02 at 10 51 23

Comment (cilippofilia) [reply content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md]: Perfect, thank you!!

**Review by yangc95 — COMMENTED** [review]
@cilippofilia I conducted a second review for this PR! Please tag me once you're done looking through them.

@SSwiniarski, is there anything else that needs improvement on your end?

**Review by yangc95 — APPROVED** [review]
@cilippofilia I made some edits here prior to approving: https://github.com/Codecademy/docs/pull/1293/commits/b9231d592533d2458b566a14b2979fc819bd908c

@SSwiniarski did you have anything else to add to this entry?

File: content/swiftui/concepts/views/terms/roundedrectangle/roundedrectangle.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```pseudo
+var body: some View {
+    RoundedRectangle(cornerRadius: Int)
+        Modifiers here
+}
+```
+
+## Example
+
+The example below displays a cyan colored rectangle, with rounded corners, with a frame of `200` wide x `50` height:
```
Comment (yangc95) [inline content/swiftui/concepts/views/terms/roundedrectangle/roundedrectangle.md:32]:
```suggestion
The example below displays a cyan colored rectangle, with rounded corners, with a frame of `200` points wide by `50` points high:
```

**Review by SSwiniarski — APPROVED** [review]
@cilippofilia, @yangc95 looks good to me.

**Comment by cilippofilia** [comment]
@yangc95 @SSwiniarski Can we finalise this?

**Comment by SSwiniarski** [comment]
Thanks @cilippofilia. Live Link is here: https://www.codecademy.com/resources/docs/swiftui/views/roundedrectangle

**Comment by Dusch4593** [comment]
@cilippofilia Thanks again for this new entry! 🚀

It looks like the image you provided isn't rendering properly:

Screen Shot 2022-12-17 at 9 17 45 AM

I think this is due to a typo in the name of the image file:

Screen Shot 2022-12-17 at 9 16 52 AM

Would you be open to opening a new PR that fixes this typo?

cc: @SSwiniarski @yangc95

**Comment by SSwiniarski** [comment]
@Dusch4593, I already pushed a PR to fix this.

**Comment by Dusch4593** [comment]
@SSwiniarski Oh cool! Just approved/merged 😄

Nevermind, @cilippofilia, sorry. Looking forward to your next Docs contribution, though! 🚀

**Pull request merged**